MonoGame.Framework/Vector4.cs

```diff
@@ -1131,6 +1143,15 @@ public override string ToString()
             return "{X:" + X + " Y:" + Y + " Z:" + Z + " W:" + W + "}";
         }
 
+        /// <summary>
+        /// Gets a <see cref="Quaternion"/> representation for this object.
+        /// </summary>
+        /// <returns>A <see cref="Quaternion"/> representation for this object.</returns>
+        public Quaternion ToQuaternion()
+        {
+            return new Quaternion(this);
+        }

```

Comment:
@KonajuGames - How do you feel about this?  The fact this has almost no code in it and is a very infrequent thing makes me feel it is an unnecessary "helper".